.github/workflows/actionlint-exceptions.txt

```diff
@@ -0,0 +1 @@
+.github/workflows/check-stylish-haskell.yml
```

Comment:
What's wrong with this workflow now?

Reply:
@carbolymer> there a number of missing quotes I believe:

```
.github/workflows/check-stylish-haskell.yml:40:7: shellcheck reported issue in this script: SC2086:info:7:66: Double quote to prevent globbing and word splitting [shellcheck]
   |
40 |       run: |
   |       ^~~~
.github/workflows/check-stylish-haskell.yml:52:7: shellcheck reported issue in this script: SC2086:info:6:22: Double quote to prevent globbing and word splitting [shellcheck]
   |
52 |       run: |
   |       ^~~~
.github/workflows/check-stylish-haskell.yml:52:7: shellcheck reported issue in this script: SC2086:info:9:26: Double quote to prevent globbing and word splitting [shellcheck]
   |
52 |       run: |
   |       ^~~~
.github/workflows/check-stylish-haskell.yml:69:7: shellcheck reported issue in this script: SC2086:info:6:22: Double quote to prevent globbing and word splitting [shellcheck]
   |
69 |       run: |
   |       ^~~~
.github/workflows/check-stylish-haskell.yml:69:7: shellcheck reported issue in this script: SC2086:info:9:26: Double quote to prevent globbing and word splitting [shellcheck]
   |
69 |       run: |
   |       ^~~~
```

I didn't fix them in this PR, because in my experience, adding checks and fixing warnings in the same PR tend to make the PR harder to merge.

I can make fix them afterwards :+1: